features_module_how_to_share_sites_config_with_oth.md

```diff
@@ -0,0 +1,121 @@
+# Features module: how to share a site’s configuration with others?
+
+If you had a chance to build one or two Drupal sites,  you probably already know that Drupal sites consists of three parts:
+**Code** (modules, themes…) stored in files, **Configuration**  (content types, views, panels…) stored in the database, and **Content** (nodes, terms…) stored in the database.
```

Comment:
did you mean it needs to write with capital letter? why did you mention just views and panels?